packages/cozy-intent/src/api/models/environments.ts

```diff
@@ -2,12 +2,35 @@ import { Connection, EventsType, MethodsType } from 'post-me'
 
 import { NativeMethodsRegister } from '../../api'
 
+export interface WebviewSourceUri {
+  uri: string
+
+  baseUrl?: never
+  html?: never
+}
+
+export interface WebiewSourceBaseUrl {
```

Comment:
```suggestion
export interface WebViewSourceBaseUrl {
```

Reply:
Woups 😅 

Thanks for the edit, but the correct form should be `Webview` and not `WebView` as the others types from this file are using this form.